src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/_help.py

```diff
@@ -8,441 +8,418 @@
+    short-summary: (PREVIEW) Manage Data Lake Analytics accounts, jobs, and catalogs.
 """
 
 helps['dla job'] = """
     type: group
-    short-summary: Commands to manage Data Lake Analytics jobs.
-    long-summary: These commands are in preview.
+    short-summary: (PREVIEW) Manage Data Lake Analytics jobs.
 """
 
 helps['dla job submit'] = """
     type: command
-    short-summary: submits the job to the Data Lake Analytics account.
+    short-summary: Submit a job to a Data Lake Analytics account.
     parameters:
         - name: --job-name
           type: string
-          short-summary: 'Job name for the job'
+          short-summary: Name for the submitted job.
         - name: --script
           type: string
-          short-summary: 'The script to submit'
-          long-summary: This is either the script contents or use `@<file path>` to load the script from a file
+          short-summary: Script to submit. This may be an @{file} to load from a file.
         - name: --runtime-version
-          short-summary: 'The runtime version to use'
+          short-summary: The runtime version to use.
           long-summary: This parameter is used for explicitly overwriting the default runtime. It should only be done if you know what you are doing.
         - name: --degree-of-parallelism
-          short-summary: 'The degree of parallelism for the job'
-          long-summary: Higher values equate to more parallelism and will usually yield faster running jobs, at the cost of more AUs consumed by the job.
+          short-summary: The degree of parallelism for the job.
+          long-summary: Higher values equate to more parallelism and will usually yield faster running jobs, at the cost of more AUs.
         - name: --priority
-          short-summary: 'The priority of the job'
+          short-summary: The priority of the job.
           long-summary: Lower values increase the priority, with the lowest value being 1. This determines the order jobs are run in.
 
 """
 
 helps['dla job cancel'] = """
     type: command
-    short-summary: cancels the job in the Data Lake Analytics account.
+    short-summary: Cancel a Data Lake Analytics job.
 """
 
 helps['dla job show'] = """
     type: command
-    short-summary: Retrieves the job in the Data Lake Analytics account.
+    short-summary: Get information for a Data Lake Analytics job.
 """
 
 helps['dla job wait'] = """
     type: command
-    short-summary: Waits for the job in the Data Lake Analytics account to finish, returning the job once finished
+    short-summary: Wait for a Data Lake Analytics job to finish.
+    long-summary: This command exits when the job completes.
     parameters:
         - name: --job-id
           type: string
-          short-summary: 'Job ID for the job to poll'
+          short-summary: 'Job ID to poll for completion.'
```

Comment:
nit: Should this full stop be here? Looks like it's not needed.

Reply:
Style is to end each description with a full stop for CLI help.